packages/terra-cell-grid/src/CellGrid.module.scss

```diff
@@ -0,0 +1,44 @@
+:local {
+  .cell-grid {
+    display: flex;
+    flex: 1 1 auto;
+    margin-bottom: 0;
+    overflow: hidden;
+    position: relative;
+
+    > * {
+      border-right: 1px solid transparent;
+
+      &:last-of-type {
+        border-right: 0;
+      }
+    }
+  }
+
+  .divider-horizontal,
+  .divider-both {
+    > * {
+      border-right: var(--terra-cell-grid-cell-border-right, 1px solid #ccc);
+
+      &:last-of-type {
+        border-right: 0;
+      }
+    }
+  }
+
+  .divider-vertical,
+  .divider-both {
+    border-top: var(--terra-cell-grid-row-border-top, 1px solid #dedfe0);
```

Comment:
Should this be `--terra-cell-grid-cell-border-top` to match the naming above with `--terra-cell-grid-cell-border-right`?